src/components/common/DataUseVoteSummary/DataUseVoteSummary.js

```diff
@@ -84,8 +85,7 @@ export default function DataUseVoteSummary({dataUseBuckets, isLoading}) {
     }, [
       rowTemplate(chunkedBuckets),
       h(ReactTooltip, {
-        id: 'tip_mixed_result',
-        place: 'left',
+        place: 'bottom',
         effect: 'solid',
         multiline: true,
         className: 'tooltip-wrapper'
```

Comment:
I'm seeing tooltips twice, one on the bottom, one on the left. This only happens on the Chair Vote tab.

<img width="431" alt="Screen Shot 2022-05-10 at 9 09 43 AM" src="https://user-images.githubusercontent.com/116679/167636148-7ddbfd52-561c-462c-8dfd-ef9006af420d.png">